doc/users/next_whats_new/2019-08_tac.rst

```diff
@@ -0,0 +1,6 @@
+
+Add ``cm.deregister_cmap`` function
```

Comment:
unregister_cmap would match the https://docs.python.org/3/library/atexit.html api (register/unregister).

Reply:
fair, done.